blacklight-cornell/app/controllers/browse_controller.rb

```diff
@@ -234,17 +113,15 @@ def redirect_catalog
     end
   end
 
- # Functions for dashboard functionality
- def check_author_status
-   base_solr = Blacklight.connection_config[:url].gsub(/\/solr\/.*/,'/solr')
-   dbclnt = HTTPClient.new
-   p =  {"q" => '"' + params[:authq].gsub("\\"," ").gsub('"',' ')+'"' }
-   woof = base_solr + "/" + @@browse_index_author + "/browse?&wt=json&" + p.to_param
-   resp = dbclnt.get_content(base_solr + "/" + @@browse_index_author + "/browse?&wt=json&" + p.to_param)# + '&' + start.to_param )
-   results = JSON.parse(resp)
-   return true if results["response"]["numFound"] == 1
-   return false if results["response"]["numFound"] != 1
- end
+  # Functions for dashboard functionality
+  def check_author_status
+    return false unless params[:authq].present?
+
+    q = '"' + params[:authq].gsub("\\"," ").gsub('"',' ') + '"'
+    results = solr_for_browse('Author').get 'browse', :params => { :q => q, :wt => :ruby }
+    return true if results['response']['numFound'] == 1
+    return false if results['response']['numFound'] != 1
```

Comment:
These two lines could be reduced to just `results['response']['numFound'] == 1` (no explicit boolean or return needed)

Reply:
I had to hold myself back from changing too much :( :(. But since you called it out, I'm going to do it!

Reply:
Changed this and a couple other methods in BrowseController that were doing similar return true/false statements: https://github.com/cul-it/blacklight-cornell/pull/2254/commits/939b191f4328207f04c2369cd9e524cf7d48caa8.

Will merge this if Jenkins builds green.